polygeist/test/polygeist-opt/red.mlir

```diff
@@ -140,4 +140,23 @@ module  {
     }  
     return
   }
+
+  // COM: Ensure reduction is not detected (exist other may alias operations).
```

Comment:
Nit: Add a comment specifying which operations form the may-alias.

Reply:
done.